wordpress-mobile/WordPress-Android
# Issue/6528 media picker insert lag

Fixes #6528 - reduces both the actual and apparent lag when inserting multiple images from the media picker. 

The lag was caused by saving the post after each image was inserted, and by optimizing images on the main thread (which blocked the editor from showing the inserted images until they were all inserted).

We now postpone saving until after all images are inserted, and perform optimization (and rotation, etc) in the background so the editor can show each image as it's inserted.

To test:

* In the `develop` branch create a post and add a dozen images to it, then notice the lag before they show up
* Do the same thing in this branch and notice images appear much more quickly
